pynipap/pynipap.py

```diff
@@ -100,7 +100,7 @@
 
         # Fetch VRF with ID 1 and print its name
         vrf = VRF.get(1)
-        print vrf.name
+        print(vrf.name)
```

Comment:
This was a silly comment.. the print() line is in the docstrings at the top of the file. Naturally we don't have print statements in pynipap.. it's a lib after all ;)